cdap-app-templates/cdap-etl/cdap-data-pipeline-base/src/main/java/io/cdap/cdap/datapipeline/SmartWorkflow.java

```diff
@@ -151,9 +155,16 @@ public class SmartWorkflow extends AbstractWorkflow {
   private int connectorNum;
   private int publisherNum;
 
+  // Keys for pipeline configs
+  public static final String PIPELINE_CONFIG_PREFIX = "system.app.pipeline.";
```

Comment:
Generally runtime arguments / preferences that started with `system.` are defined and understood by the CDAP platform, which shouldn't be app specific. Why do we want `system.` prefix in this case?

Reply:
will change